### PR TITLE
chore: add new issue template for CDE implementation requests

### PR DESCRIPTION
**Pull request by e-t-k**
**PR Close Date:** October 18

Added a new issue template for implementing newly approved CDES:
https://github.com/CBIIT/ccdi-federation-api/blob/chore/new-cde-issue-template/.github/ISSUE_TEMPLATE/new_cde.md
 
Any additional elements we want to track? (Or conversely, are any of these too duplicative?)

### CODE REVIEW

**Review by claymcleod — APPROVED** [review]
Looks good!

**Review by grugna — APPROVED** [review]

**Comment by e-t-k** [comment]
Thanks! I will merge this after #118 is merged.

**Pull request merged**
